PocketKit/Sources/Sync/V3/V3Client.swift

```diff
@@ -111,12 +111,18 @@ public class V3Client: NSObject, V3ClientProtocol {
             }
 
             let decoder = JSONDecoder()
-            decoder.keyDecodingStrategy = .convertFromSnakeCase
-            guard let response = try? decoder.decode(decodable, from: data) else {
+            decoder.keyDecodingStrategy = .useDefaultKeys
+            do {
+                let str = String(decoding: data, as: UTF8.self)
+                print(str)
+                let response = try decoder.decode(decodable, from: data)
+                return response
+            } catch {
+                print(String(describing: error))
```

Comment:
is this print still necessary since we're also logging it?

Reply:
i was getting different errors with the descriptor. ill log one and remove the other.